// (c) Nick Polyak 2023 
// License: MIT License (https://opensource.org/licenses/MIT)
//
// short overview of copyright rules:
// 1. you can use this framework in any commercial or non-commercial 
//    product as long as you retain this copyright message
// 2. Do not blame the author of this software if something goes wrong. 
// 
// Also, please, mention this software in any documentation for the 
// products that use it.

syntax = "proto3";

package NP.Gidon.Messages;

enum Topic
{
	None = 0;
	WindowInfoTopic = 1;
}

message WindowInfo
{
	string UniqueWindowHostId = 1;
	int64 WindowHandle = 2;
}
